src/model/cloud-runner/workflows/async-workflow.ts

```
@@ -27,7 +27,16 @@ printenv
 git config --global advice.detachedHead false
 git config --global filter.lfs.smudge "git-lfs smudge --skip -- %f"
 git config --global filter.lfs.process "git-lfs filter-process --skip"
-git clone -q -b ${CloudRunner.buildParameters.cloudRunnerBranch} ${CloudRunnerFolders.unityBuilderRepoUrl} /builder
+BRANCH="${CloudRunner.buildParameters.cloudRunnerBranch}"
+REPO="${CloudRunnerFolders.unityBuilderRepoUrl}"
+if [ -n "$(git ls-remote --heads \"$REPO\" \"$BRANCH\" 2>/dev/null)" ]; then
```

ESLint

[error] Unnecessary escape character: \".

(no-useless-escape)